components/match2/commons/match_group_display.lua

```diff
@@ -102,6 +102,34 @@ function MatchGroupDisplay.MatchGroupById(args)
 	})
 end
 
+--[[
+Displays a singleMatch specified by a bracket ID and matchID.
+]]
+function MatchGroupDisplay.MatchByMatchId(args)
+	local bracketId = args.id
+	local matchId = args.matchid
+	assert(bracketId, 'Missing bracket ID')
+	assert(matchId, 'Missing match ID')
+
+	matchId = MatchGroupUtil.matchIdFromKey(matchId)
+
+	local matchGroup = MatchGroupUtil.fetchMatchGroup(bracketId)
+	local fullMatchId = bracketId .. '_' .. matchId
+	local match = matchGroup.matchesById[fullMatchId]
+
+	assert(match, 'Match bracketId= ' .. bracketId .. ' matchId=' .. matchId .. ' not found')
+
+	local SingleMatchDisplay = Lua.import('Module:MatchGroup/Display/SingleMatch', {requireDevIfEnabled = true})
+	local config = SingleMatchDisplay.configFromArgs(args)
+
+	local MatchGroupContainer = require('Module:Brkts/WikiSpecific').getMatchContainer('singleMatch')
```

Comment:
Would be preferred to have these imports up top

Reply:
SingleMatchDisplay was put here to reduce module include size, since the other entry points don't use it.

require('Module:Brkts/WikiSpecific') should be moved up top

Reply:
Moved WikiSpecific to the top. Also adjusted the 3 other places where WikiSpecific was imported in this module.